tests/hypothesis_strategies/targets.py

```diff
@@ -5,28 +5,28 @@
 from baybe.targets.enum import TargetMode
 from baybe.targets.numerical import _VALID_TRANSFORMATIONS, NumericalTarget
 
-from .utils import interval
+from .utils import intervals as st_intervals
 
 target_name = st.text(min_size=1)
 """A strategy that generates target names."""
 
 
 @st.composite
-def numerical_target(draw: st.DrawFn):
+def numerical_targets(draw: st.DrawFn, intervals=None):
```

Comment:
i think we have not been super accurate with type annotations in these strategies

eg here im not even sure whats the intended parameter type? If its a single given interval, this parameter should prob even be renamed to `interval` not `intervals`, but maybe you intend this to be a strategy?

Reply:
Yes, it's supposed to be a strategy for having the option to narrow the target strategy (was needed for this PR). Agree, so far the type annotations are not very rigorous in the tests, and I omitted it here simply because it was not obvious to me what the type should be. But simply the fact that you asked shows that we should be a bit more rigorous here, even if it just helps to better understand what's going on. Will try to fix this, perhaps `reveal_type` can help me here. Also, can give a more suggestive parameter name. Leaving this thread open until I find a solution...

Reply:
Looking at the import, I would interpret that `intervals` here refers to either a single interval or a colletion of those, since the strategy is probably `st_intervals`. But I fully agree, having more informative type hints would be great and very much appreciated also from me 😄